kong/db/schema/entities/routes.lua

```diff
@@ -3,6 +3,18 @@ local router = require("resty.router.router")
 local deprecation = require("kong.deprecation")
 
 local CACHED_SCHEMA = require("kong.router.atc").schema
+local _get_expression = require("kong.router.compat")._get_expression
```

Comment:
`_` is usually only for tests. If it is a API schema uses, it should not have the `_` in front.

Reply:
done